src/coreComponents/finiteVolume/SurfaceElementStencil.hpp

```diff
@@ -44,48 +44,14 @@ namespace geosx
 /// @endcond
 
 /**
- * @struct SurfaceElementStencil_Traits
- * Struct to predeclare the types and constexpr values of SurfaceElementStencil so that they may be used in
- * StencilBase.
+ * @brief Describes properties of SurfaceElementStencil.
  */
-struct SurfaceElementStencil_Traits
-{
-  /// The array type that will be used to store the indices of the stencil contributors
-  using IndexContainerType = ArrayOfArrays< localIndex >;
-
-  /// The array view type for the stencil indices
-  using IndexContainerViewType = ArrayOfArraysView< localIndex >;
-
-  /// The array view to const type for the stencil indices
-  using IndexContainerViewConstType = ArrayOfArraysView< localIndex const >;
-
-  /// The array type that is used to store the weights of the stencil contributors
-  using WeightContainerType = ArrayOfArrays< real64 >;
-
-  /// The array view type for the stencil weights
-  using WeightContainerViewType = ArrayOfArraysView< real64 >;
-
-  /// The array view to const type for the stencil weights
-  using WeightContainerViewConstType = ArrayOfArraysView< real64 const >;
-
-  /// Number of points the flux is between (normally 2)
-  static localIndex constexpr NUM_POINT_IN_FLUX = 6;
-
-  /// Maximum number of points in a stencil
-  static localIndex constexpr MAX_STENCIL_SIZE = 6;
-
-  /// Maximum number of connections in a stencil
-  static localIndex constexpr MAX_NUM_OF_CONNECTIONS = MAX_STENCIL_SIZE * (MAX_STENCIL_SIZE - 1) / 2;
-};
+using SurfaceElementStencilTraits = StencilTraits< ArrayOfArrays, 6, 6, 6 * ( 6 - 1) / 2 >;
```

Comment:
Maybe we should add a small comment here to remind future developers that `6 * ( 6 - 1) / 2` is really `maxStencilSize * (maxStencilSize - 1) / 2`. That was self-documented by the variable names in the previous implementation, but that this information may be lost with the new `localIndex` templates

Reply:
Agreed, I'm expanding the documentation for this type alias.